tests/remote_attestation/maa/run_scenario.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
+
+# In this scenario we have two enclaves: sgxlkl_enclave and oe_enclave. 
+# oe_enclave is the tls server and runs and waits 120 seconds 
+# sgxlkl_enclave starts and connects to oe_enclave
+# sgxlkl_enclave gets the report from oe_enclave and then gets authentication token from AAD
+# sgxlkl_enclave calls MAA for JWT token and gets, parses and verifies it 
+
+# Run mode is sw or hw
+export SGXLKL_RUN_MODE=$1
+
+# Verify that dependent env variables are available
+if [[ -z $SGXLKL_PREFIX || -z $SGXLKL_ROOT || -z $SGXLKL_RUN_MODE || -z $MAA_CLIENT_ID || -z $MAA_CLIENT_SECRET  || -z $MAA_APP_ID || -z $MAA_ADDR || -z $MAA_ADDR_APP ]]; then
+    echo "Dependent environment variables for this scenario not available."
+    echo "SGXLKL_ROOT: $SGXLKL_ROOT"
+    echo "SGXLKL_PREFIX: $SGXLKL_PREFIX"
+    echo "SGXLKL_RUN_MODE: $SGXLKL_RUN_MODE"
+    exit 1
+fi
+
+# If openenclave is not installed, install it
+if [[ ! -d "/opt/openenclave" ]]; then
+    home=$(pwd)
+    cd ~ || exit 1
+    sudo rm -rf openenclave/
+    git clone --recursive -b feature/sgx-lkl-support https://github.com/openenclave/openenclave.git
```

Comment:
This is not sustainable. We would have to install OE repeatedly if we add more tests that need it. @Pengpeng-Microsoft Can you recommend something?

Reply:
That is not true @letmaik 
if you look at containing if block, it will only do that if /opt/openenclave doesn't exits

```
if [[ ! -d "/opt/openenclave" ]]; then
```

Reply:
OK, I see it now. I don't think that's good though as you will never update the OE version then if a CI VM is not torn down. It would be better to install OE in a folder relative to the sgx-lkl repository, like sgx-lkl itself does it. It still doesn't seem right to have all this heavy setup machinery in this particular test.

Reply:
@Pengpeng-Microsoft @jxyang @paulcallen any idea what is the best solution here? To me it is acceptable if OE SDK is not updated with every run but it is also good to use the latest OE SDK. How should we proceed about that?

Reply:
@letmaik: the heavy machinery is fine here, but we should be careful to reuse all this in future tests. Applications will have to bring their own libraries, in this case it just happens to be a (user-mode) OE. 

Definitely don't install anything system-wide, always locally. The `git clone` here will always get the latest version, i.e. this can change at random points in time, so don't keep any state, otherwise it will be hell to figure out why the CI machine behaves differently to our dev machines.